components/Layout.vue

```diff
@@ -11,7 +11,7 @@
       >
     </div>
     <div class="relative mx-auto desktop-gutters flex justify-center">
-      <div class="min-w-0 max-w-2xl flex-auto px-8 pb-16 pt-8 lg:max-w-none">
+      <div class="min-w-0 max-w-2xl flex-auto px-8 pb-16 pt-8 xl:pt-16 lg:max-w-none">
```

Comment:
The padding has only been adjusted for when the table of contents is visible (pt-16 doesn't look great on mobile)